README.md

```diff
@@ -121,25 +131,34 @@ optional arguments:
                         the DHCP. (default: nat)
 ```
 
-# virtualize.sh
+## virtualize.sh
 
 `virtualize.sh` is a script built on top of `virtualizor.py` to play SpinalStack deployment and upgrade.
 
 ```sh
-$ ./virtualize.sh --help
-usage: virtualize.sh [OPTION] workdir1 workdir2 etc
+usage: ./virtualize.sh [OPTION]
 Collect architecture information from the edeploy directory as generated
-by config-tools/download.sh.
+by config-tools/download.sh and use the virtulizor.py to boostrap a SpS platform.
 
-optinal arguments:
-  -H|--hypervisor=name: change the hypervisor name, default ()
 arguments:
-virtualize.sh will use the first argument as the location of the
-SpinalStack environment to deploy. It will then upgrade the newly deployed
-SpinalStack to the following environment directory.
-
-For example: ./virtualize.sh I.1.2.1/
-will deploy environment from the I.1.2.1/ directory.
+    -h|--help                     Show this help
+    -H|--hypervisor=name          Set the hypervisor hostname, default (localhost)
+    -d|--debug                    Set the debug mode for this script, default: disabled
+    -v|--virt=virt_platform.yml   Set the path to the infra's yaml, default: (virt_platform.yml)
+    -e|--extra='--cleanup'        Add extra parameters to virtulizor.py
+    -p|--prefix                   Change the platform's prefix, default: unix user
+    -s|--socks                    Create a socks server to test your platform
+    -t|--tempest                  Launch the sanity job at the end of a deployement
+    -l|--logs                     Fetch the logs from elasticesearch and upload them
+                                  into Swift
+
+For example:
+./virtualize.sh -H localhost -e '--clean' I.1.2.1/
```

Comment:
You specify cleanup at line 148 and you use clean as extra param to virtualizor.py